modin/backends/pandas/query_compiler.py

```diff
@@ -835,6 +835,151 @@ def resample_var(self, resample_args, ddof, *args, **kwargs):
     def resample_quantile(self, resample_args, q, **kwargs):
         return self._resample_func(resample_args, "quantile", q=q, **kwargs)
 
+    def rolling_win_mean(self, rolling_args, *args, **kwargs):
```

Comment:
```suggestion
    rolling_win_mean = FoldFunction.register(
        lambda df, rolling_args, *args, **kwargs: df.rolling(*rolling_args).mean(*args, **kwargs)
    )
```

Reply:
Is it possible to rewrite all of these like this?

Reply:
I rewrote all possible functions in this style.